packages/theme/README.md

```diff
@@ -17,134 +17,168 @@ Documented in [dooboo-ui.dooboolab.com](https://dooboo-ui.dooboolab.com/?path=/d
+        label: colors.gray6,
+        placeholder: colors.gray5,
+        disabled: colors.gray3,
+        validation: colors.red2,
+        contrastBasic: colors.white,
+        contrastLabel: colors.gray3,
+        contrastPlaceholder: colors.gray4,
+        contrastDisabled: colors.gray6,
+        contrastValidation: colors.red1,
     },
-    light: {
-      text: '#000000',
-      bg: colors.light,
+    button: {
+        primary: {
+            text: colors.white,
+            bg: colors.primaryLight,
+        },
+        secondary: {
+            text: colors.white,
+            bg: colors.secondaryLight,
+        },
+        success: {
+            text: colors.white,
+            bg: colors.successLight,
+        },
+        warning: {
+            text: colors.white,
+            bg: colors.warningLight,
+        },
+        danger: {
+            text: colors.white,
+            bg: colors.dangerLight,
+        },
+        info: {
+            text: colors.white,
+            bg: colors.infoLight,
+        },
+        light: {
+            text: colors.black,
+            bg: colors.gray1,
+        },
+        disabled: {
+            text: colors.gray3,
+            bg: colors.gray1,
+        },
     },
-  },
 };
 
 export const dark: typeof light = {
-  bg: {
-    default: '#000000',
-    paper: '#414141',
-    disabled: '#C4C4C4',
-    border: 'rgba(255,255,255,0.2)',
-  },
-  role: {
-    primary: colors.darkPrimary,
-    secondary: colors.secondary,
-    danger: colors.danger,
-    warning: colors.warning,
-    info: colors.info,
-    success: colors.success,
-    light: colors.light,
-  },
-  text: {
-    default: '#FFFFFF',
-    primary: '#000000',
-    placeholder: '#6D6D6D',
-    disabled: '#C4C4C4',
-    contrast: '#000000',
-    validation: '#FF002E',
-    secondary: '#D3D8E8',
-    link: '#E0E0E0',
-    accent: '#8A96DC',
-  },
-  button: {
-    primary: {
-      text: '#FFFFFF',
-      bg: colors.darkPrimary,
+    bg: {
+        basic: colors.black,
+        paper: colors.gray9,
+        disabled: colors.gray7,
+        border: colors.gray6,
+        borderContrast: colors.gray2,
     },
-    secondary: {
-      text: '#000000',
-      bg: colors.secondary,
+    role: {
+        primary: colors.primaryDark,
+        secondary: colors.secondaryDark,
+        accept: colors.acceptDark,
+        info: colors.infoDark,
+        link: colors.linkDark,
+        success: colors.successDark,
+        warning: colors.warningDark,
+        danger: colors.dangerDark,
     },
-    success: {
-      text: '#000000',
-      bg: colors.success,
+    text: {
+        basic: colors.white,
+        label: colors.gray3,
+        placeholder: colors.gray4,
+        disabled: colors.gray6,
+        validation: colors.red1,
+        contrastBasic: colors.black,
+        contrastLabel: colors.gray6,
+        contrastPlaceholder: colors.gray5,
+        contrastDisabled: colors.gray3,
+        contrastValidation: colors.red2,
```

Comment:
Let's remove everything except `contrastBasic`, and this can be renamed to `contrast`.